poetry/version/markers.py

```diff
@@ -614,6 +614,12 @@ def without_extras(self):
 
         return MarkerUnion(*new_markers)
 
+    def is_any(self):  # type: () -> bool
```

Comment:
I think I fixed/added this in `1.0.0b9` can you try again on the latest release?

PR: https://github.com/sdispater/poetry/pull/1650

Reply:
Trying immediatly!

Reply:
You may need to re-lock as the previous versions created issues with the lockfile because of that.

Reply:
Seems to be working perfectly! Thanks! :D